<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI键盘播放器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDropMode::DropOnly</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openPlaylistButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打开midi文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadFileListButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pitchHigherButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>提高八度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pitchLowerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>降低八度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pitchKeepButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保持音高</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
